### ISSUE TITLE
Arrow Rain does not proc

### ISSUE THREAD

**Issue by lolmetimbers** [Daniel12321/AdventureMMO#45]
Arrow rain does not trigger on any shots. Even with 60% chance. 2.07 API 5 1.10.2 and modded minecraft. Maybe something else is interfering?

I tested this with normal minecraft bow and armorplus bows

**Comment by Daniel12321** [comment]
Yep, this is caused by a sponge bug, and cant be fixed till that is fixed.
